src/wp-admin/includes/class-file-upload-upgrader.php

```diff
@@ -88,7 +88,7 @@ public function __construct( $form, $urlholder ) {
 							self_admin_url( 'theme-install.php' ),
 							__( 'Return to the Theme Installer' )
 						);
-						wp_die( __( 'Incompatible Archive.' ) . '<br />' . $themes_page );
+						wp_die( __( 'Incompatible Archive.' ) . '<br />' . $themes_page, 422 );
```

Comment:
Same question here about 415.

```suggestion
						wp_die( __( 'Incompatible Archive.' ) . '<br />' . $themes_page, 415 );
```

Reply:
Same comment as on line 82 above